src/Library/demos/User Info/main.js

```diff
@@ -0,0 +1,48 @@
+import Xdp from "gi://Xdp";
+import XdpGtk from "gi://XdpGtk4";
+import Gio from "gi://Gio";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+
+const portal = new Xdp.Portal();
+const parent = XdpGtk.parent_new_gtk(workbench.window);
+
+const button = workbench.builder.get_object("button");
+const avatar = workbench.builder.get_object("avatar");
+const entry = workbench.builder.get_object("entry");
+const username = workbench.builder.get_object("username");
+const display = workbench.builder.get_object("name");
+
+Gio._promisify(
+  Xdp.Portal.prototype,
+  "get_user_information",
+  "get_user_information_finish",
+);
+
+async function onClicked() {
+  // call the portal function
+  const reason = entry.get_text();
+  let user_info = null;
+  user_info = await portal.get_user_information(parent, reason, null, null);
+
+  //retrieve values from GVariant
+  user_info = user_info.deepUnpack();
```

Comment:
I really like the gvariant description comment you made in the other PR, I would do something similar here

```suggestion
  const result = await portal.get_user_information(parent, reason, null, null);
  // result is a ...
  const user_info = user_info.deepUnpack();
```